pkg/datastoresrp/api/v20231001preview/testdata/mongodatabaseresourcedatamodel_recipe.json

```diff
@@ -23,6 +23,12 @@
             }
           },
         "host": "testAccount1.mongo.cosmos.azure.com",
-        "port": 10255
+        "port": 10255,
+        "status": {
+            "recipe": {
+                "templateKind": "bicep",
```

Comment:
Do we need examples with Terraform?

Reply:
pkg/corerp/api/v20231001preview/testdata/environmentresource-terraformrecipe-localpath.json